robot_ws/src/cloudwatch_robot/src/cloudwatch_robot/monitor_distance_to_goal.py

```diff
@@ -1,65 +1,71 @@
 #!/usr/bin/env python
 """
- Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-
- Permission is hereby granted, free of charge, to any person obtaining a copy of this
- software and associated documentation files (the "Software"), to deal in the Software
- without restriction, including without limitation the rights to use, copy, modify,
- merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
- permit persons to whom the Software is furnished to do so.
-
- THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
- INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
- PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
- HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
- OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
- SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining
```

Comment:
What prompted this line break change? Is a linter complaining about line length? This would affect the change how we need to change `ament_copyright` upstream.

Reply:
Also, I realized that our licenses in Python files need to be in the format of
```
# Block of text, line 1
# Block of text, line 2
# Block of text, line 3
```
instead of
```
"""
Block of text, line 1
Block of text, line 2
Block of text, line 3
"""
```
to make `ament_copyright` happy.

Reply:
Looking at this [source file](https://github.com/johann-petrak/licenseheaders/blob/master/licenseheaders.py) for example, `"""` and `#` do seem to mean different things.